conans/client/graph/range_resolver.py

```diff
@@ -6,18 +6,34 @@
 def satisfying(list_versions, versionexpr, output):
     """ returns the maximum version that satisfies the expression
     if some version cannot be converted to loose SemVer, it is discarded with a msg
-    This provides some woraround for failing comparisons like "2.1" not matching "<=2.1"
+    This provides some workaround for failing comparisons like "2.1" not matching "<=2.1"
     """
     from semver import SemVer, max_satisfying
-    version_range = versionexpr.replace(",", " ")
+    version_range = versionexpr.split(",")
+
+    include_prerelease = False
+    if "include_prerelease=True" in version_range:
```

Comment:
I'd say this can be simplified, and assume "prerelease" means enabling them (nothing to keep current behavior)

Reply:
Changed to `prerelease`